repo/meta/schema/package-schema.json

```diff
@@ -22,25 +22,32 @@
       "properties": {
         "packagingVersion": {
           "type": "string",
-          "enum": ["2.0"]
+          "enum": ["2.0"],
+          "description": "The version of the packaging format. Please do not use this version. It is only here for backwards compatibility. We recommend you use the latest version."
         },
         "name": {
-          "type": "string"
+          "type": "string",
+          "description": "The name of the package."
         },
         "version": {
-          "type": "string"
+          "type": "string",
+          "description": "The version of the package. We recommend you use semantic versions."
         },
         "scm": {
-          "type": "string"
+          "type": "string",
+          "description": "Software configuration management. Location where can users view information, and file bugs."
```

Comment:
I think this is actually the *other* SCM: [source control management](https://en.wikipedia.org/wiki/Source_control_management), since it's usually the URL of the project's GitHub repo.